.github/workflows/conformance-k8s-kind.yaml

```diff
@@ -196,6 +196,16 @@ jobs:
             --report-dir=${E2E_REPORT_DIR}                \
             --disable-log-dump=true
 
+      - name: Poor man's file-descriptor leak detector
+        run: |
+          for p in $(pidof cilium-agent); do
+            num=$(sudo lsof -p $p | wc -l)
+            if [ "$num" -gt "500" ]; then
+              echo "cilium-agent($p) has more than 500 file descriptors (potential leak)"
```

Comment:
Apologies for the drive-by commit - but if implementable I think this would be a good Prometheus metric to have (which we could then also assert in `cilium connectivity check` in all workflows)

Reply:
I moved the commits to https://github.com/cilium/cilium/pull/36241.